lib/delegate-electron-events.js

```diff
@@ -0,0 +1,47 @@
+const EventEmitter = require('events').EventEmitter
+const electron = require('electron')
+
+const ipcMain = electron.ipcMain
+const app = electron.app
+
+module.exports = delegateElectronEvents
+
+// delegate electron events while booting
+// until a window is ready that can handle them
+// null -> EventEmitter
```

Comment:
we're not using those api comments in this project yet, either we should add them everywhere or stick to the coding style